src/styles/blocks/explore.scss

```diff
@@ -0,0 +1,56 @@
+.explore {
+  &__item {
+    margin-bottom: 48px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    @media onTablet {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.explore-option {
+  &__photo {
+    width: 100%;
+    object-fit: center;
+    object-position: center;
+
```

Comment:
remove space

Reply:
thank you for the correction, it have been fixed